crates/wick/wick-runtime/src/triggers/http.rs

```diff
@@ -265,7 +265,10 @@ fn register_static_router(
       ))
     }
   };
-  let router = StaticRouter::new(volume, Some(router_config.path().to_owned()));
+
+  let fallback = router_config.fallback().map(|s| s.clone());
```

Comment:
this can be `.fallback().clone()` no?

Reply:
fallback can be `None`. So this should return either the fallback string or None?